tests/helpers/mobiledoc.js

```diff
@@ -0,0 +1,26 @@
+import PostNodeBuilder from 'content-kit-editor/models/post-node-builder';
+import MobiledocRenderer from 'content-kit-editor/renderers/mobiledoc';
+
+/*
+ * makeMD({post, section, marker, markup} =>
+     post([
+       section('P', [
+         marker('some text', [markup('B')])
+       ])
+     })
+ */
+function makeMD(treeFn) {
+  let builder = new PostNodeBuilder();
+
+  const post    = (...args) => builder.createPost(...args);
+  const section = (...args) => builder.createMarkupSection(...args);
```

Comment:
ha, I'll change it. I always consider card sections to be special types of sections. I'm going to have to make a vim macro to cut down on the typing. :)